test/unit/io_managers_test.py

```diff
@@ -167,3 +171,43 @@ def test_missing_schema_error(sqlite_io_manager_fixture):
     output_context = build_output_context(asset_key=AssetKey(asset_key))
     with pytest.raises(RuntimeError):
         manager.handle_output(output_context, venue)
+
+
+@pytest.fixture
+def pudl_sqlite_io_manager_fixture(tmp_path):
+    """Create a SQLiteIOManager fixture with a PUDL database schema."""
+    md = MetaData()
+    artist = Table(  # noqa: F841
+        "artist",
+        md,
+        Column("artistid", Integer, primary_key=True),
+        Column("artistname", String(16), nullable=False),
+    )
+    track = Table(  # noqa: F841
+        "track",
+        md,
+        Column("trackid", Integer, primary_key=True),
+        Column("trackname", String(16), nullable=False),
+        Column("trackartist", Integer, ForeignKey("artist.artistid")),
+    )
+    manager = PudlSQLiteIOManager(base_dir=tmp_path, db_name="pudl")
+    # Override the default PUDL metadata with this test metadata, so that they
+    # don't match, and we can check what happens.
+    manager.md = md
```

Comment:
This seems janky. What's the right way to do this?

Reply:
I think the correct way to test this is to create a fresh Package object with test data.